storage/storage-service/src/lib.rs

```diff
@@ -35,6 +35,103 @@ use storage_proto::proto::storage::{
 };
 use tokio::runtime::Runtime;
 
+use libra_secure_net::NetworkServer;
+use libra_types::{account_state_blob::AccountStateBlob, proof::SparseMerkleProof};
+use std::thread::{self, JoinHandle};
+use storage_proto::StartupInfo;
+
+/// Starts storage service with a given LibraDB
+pub fn start_simple_storage_service_with_db(
```

Comment:
can you comment a bit why we have both `start_simple_storage_service_with_db` and `start_simple_storage_service`?